enterprise/tpa_pipeline.py

```diff
@@ -78,7 +98,10 @@ def active_provider_enforces_data_sharing(request, enforcement_location):
         enforcement_location (str): the point where to see data sharing consent state.
         argument can either be "optional", 'at_login' or 'at_enrollment'
     """
-    running_pipeline = request.session.get('partial_pipeline')
+    try:
+        running_pipeline = get_partial_pipeline(request)
+    except TypeError:
+        running_pipeline = request.session.get('partial_pipeline_token')
```

Comment:
@bradenmacdonald I think this is to protect against the case when we're "NotConnectedToOpenEdX". I'm not sure if it's better to check for the dependency and remove the try-catch block, or to keep it going like this.